### ISSUE TITLE
Documentation of factors

### ISSUE THREAD

**Issue by yetongatech** [borglab/gtsam#30]
May I know if there is a documentation of all the factors? @dellaert

**Comment by dellaert** [comment]
I agree this is lacking. I have been wanting to improve doxygen main page.

**Comment by isaiahoh** [comment]
Hi @dellaert,
I've started working on improved documentation for factors on my fork of GTSAM. I'm using Sphinx connected to Doxygen in a Read the Docs format. This approach allows for better organization of related terms like Factors without affecting the current documentation structure.
The Read the Docs format with Sphinx offers several advantages:
1. Easy navigation with a sidebar for quick access to different sections
2. Search functionality for finding specific topics

I've attached a screenshot below for a first look at the new documentation structure in the Read the Docs format. I'd appreciate your feedback on this approach and the initial layout. Let me know if this direction could be valuable for the project.
<img width="1015" alt="image" src="https://github.com/user-attachments/assets/5bec5e2e-4fbf-472a-a8a1-874860643746">

**Comment by dellaert** [comment]
See discussion [here](https://github.com/borglab/gtsam/discussions/1473). Unfortunately that effort has died out, student graduating and all. However, this is at least a good user guide: https://borglab.github.io/gtsam/
